homeassistant/components/cover/myq.py

```diff
@@ -17,6 +17,7 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+DEFAULT_DEVICE_CLASS = 'garage'
```

Comment:
Please move the value into the function. The const is used only once and the name (DEFAULT_) isn't straight.